lib/fs-driver.js

```diff
@@ -98,13 +139,11 @@ FsDriver.prototype.move = function (source, dest, cb) {
  * @return {string} The copy command to be executed.
  */
 FsDriver.prototype.copy = function (source, dest, cb) {
-  var command = [
+  return this.exec([
```

Comment:
having a callback and a return value might get confusing after a while. is there a reason you don't put the return value in the `cb` ?